src/components/BookComp.js

```diff
@@ -1,31 +1,45 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import './styles/BookComp.css';
 
 function BookComp({
   bookId, title, author, category, onClick,
 }) {
   return (
-    <div className="books-container">
-      <div className="book-container">
+    <div className="book-container">
+      <div className="bookss">
         <h3 className="book-category">{category}</h3>
         <h2 className="book-title">{title}</h2>
         <h3 className="book-author">{author}</h3>
         <ul className="options">
           <li>
-            <button type="button">Comments</button>
+            <button type="button" className="button">Comments</button>
           </li>
           <li>
-            <button type="button" onClick={onClick} id={bookId}>
+            <button type="button" className="button" onClick={onClick} id={bookId}>
               Remove
             </button>
           </li>
           <li>
-            <button type="button">Edit</button>
+            <button type="button" className="button">Edit</button>
           </li>
         </ul>
       </div>
       <div className="book-prog">
-        <div className="book-container">
+        <div className="progress-section">
+          <progress
+            value="75"
+            min="0"
+            max="100"
+            style={{
+              height: 0,
+              width: 0,
+            }}
+          >
+            75%
+          </progress>
+        </div>
+        <div className="book-container2">
```

Comment:
- [ ] @siddghosh108 Great work styling the app, however,  the thickness is this section is bigger than in the design, check the comparison below:


Zeplin Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/siddghosh108/Book-Store-Redux/assets/61408860/2666aece-c579-40d5-8d30-945587c3397b)  | ![image](https://github.com/siddghosh108/Book-Store-Redux/assets/61408860/1aa30988-a379-4d29-a493-3af50d1ac3ca)

Reply:
I have decreased thethickness of the progressbar the way you wanted. Please see below:
![Uploading image.png…]()